js/ad_manager_vast.js

```diff
@@ -776,10 +1114,10 @@ OO.Ads.manager(function(_, $) {
      * The xml is parsed to find any tracking events and then returned as part of an object.
      * @private
      * @method Vast#parseTrackingEvents
-     * @param {array} tracking to add the tracking info to and return.
-     * @param {xml} xml The data of the ad with tracking info.
-     * @param {array} trackingEvents List of events that are tracked, if null then it uses the global one.
-     * @returns {array} tracking An array of tracking items.
+     * @param {array} tracking to add the tracking info to and return
+     * @param {xml} xml The data of the ad with tracking info
+     * @param {array} trackingEvents List of events that are tracked, if null then it uses the global one
+     * @returns {array} An array of tracking items.
```

Comment:
Yeah "array" definitely isn't supported. You can actually just add "[]" to a type and that will connote that it is an array. So in this case, "tracking" is an array of strings so the syntax should be "{string[]}". I'll make this change for the rest of them. Here's the docs @stephanieblotner pointed me to: http://usejsdoc.org/tags-type.html